packages/contentful/src/mocks/graphql-client.mock.ts

```diff
@@ -17,12 +17,16 @@ export const getContentfulGraphqlClientMockServer = (
   });
   const mocks = {
     JSON: () => '2021-10-12T15:42:05Z',
+    DateTime: () => null,
```

Comment:
For reasons I cannot fathom this line shows as not covered, but also the tests fail completely if it is removed.

Reply:
yeah to be fair i wouldn't mind skipping coverage checks for this file